src/Application/Command/CreateMilestones.php

```diff
@@ -41,14 +41,12 @@ public function execute(InputInterface $input, OutputInterface $output): int
         return 0;
     }
 
-    private function createMilestoneIfNotExists(RepositoryName $repositoryName, SemVerVersion $version): bool
+    private function createMilestoneIfNotExists(RepositoryName $repositoryName, SemVerVersion $version): void
```

Comment:
i believe `bool` was used to avoid the `UnusedMethodCall` psalm issue, but i think it's better to suppress it, can be reverted.

Reply:
Let's suppress it locally, rather than suppressing it globally in `psalm.xml`